projects/smoothing-algos-python.yml

```
@@ -0,0 +1,32 @@
+---
+name: Developing a Pythonic tool for smoothing histograms
+postdate: 2025-02-23
+categories:
+  - Analysis tools
+  - Open science
+durations:
+  - 3 months
+experiments:
+  - ATLAS
+  - CMS
+  - HLLHC
+skillset:
+  - Python
+status:
+  - Available
+project:
+  - IRIS-HEP
+location:
+  - Any
+commitment:
+  - Full time
+program:
+  - IRIS-HEP fellow
+shortdescription: Developing a Pythonic tool for smoothing histograms
+description: Histograms are fundamental objects in statistical modelling frameworks used in many Large Hadron Collider (LHC) analyses. They serve as the primary representation of both experimental data and theoretical predictions, with systematic uncertainties often modelled as variations on these histograms. Smoothing refers to techniques that reduce statistical noise in histograms, enhancing the stability of statistical models and ultimately leading to more reliable results in physics analyses. While smoothing is essential for analyses, existing tools lack accessible interfaces, comprehensive documentation, and  integration into the  Scikit-HEP ecosystems. This project aims to address this gap by developing a smoothing tool in Python based on existing algorithms, enabling physicists to apply smoothing methods more efficiently and consistently in their workflows.
+
+contacts:
+  - name: Mohamed Aly
+    email: mohamed.aly@cern.ch
+  - name: Alexander Held
+    email: alexander.held@cern.ch
```

```suggestion
  - name: Alexander Held
    email: alexander.held@cern.ch
  - name: Matthew Feickert
    email: matthew.feickert@cern.ch
```

While I don't have prior knowledge of the TRexFitter tool, I know that Alex is already going to be mentoring other Fellows for the AGC, so I would be happy to be co-listed here to help out where I can.